AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -810,20 +825,24 @@ - (void)reloadRowsAtIndexPaths:(NSArray *)indexPaths withAnimationOptions:(ASDat
     
     // Reloading requires re-fetching the data.  Load it on the current calling thread, locking the data source.
     [self accessDataSourceWithBlock:^{
-      NSMutableArray *nodes = [[NSMutableArray alloc] initWithCapacity:indexPaths.count];
+      NSMutableArray<ASIndexedNodeContext *> *contexts = [[NSMutableArray alloc] initWithCapacity:indexPaths.count];
       
       // FIXME: This doesn't currently do anything
```

Comment:
I was thinking the same thing. We can _definitely_ remove it since it literally doesn't do anything right now except waste time.

As for the deeper question about all the corresponding sorts (whose results are actually used (!)), it's a little tricky. Since in practice we always use `ASChangeSetDataController`, all our changes should already be received in safe order, so all the sorts are redundant (or maybe even dangerous i.e. sorting deletes ascending) but _technically_ ASDataController the superclass was not designed with this guarantee in mind.